src/cli/index.js

```diff
@@ -8,7 +8,7 @@ import { existsFileSync } from '../util/exists';
 import parseConfig from './parseConfig';
 
 const cliOptions = parseArgv(process.argv.slice(2), true);
-const configOptions = parseConfig();
+const configOptions = parseConfig(cliOptions.opts);
 const defaultOptions = parseArgv([]);
 
 const options = _.defaults({}, cliOptions, configOptions, defaultOptions);
```

Comment:
`cliOptions.opts` is only defined when provided via `--opts`. Without this argument opts is undefined and the fallback to mocha-webpack.opts will no longer work.

Should be rewritten as
```javascript
const cliOptions = parseArgv(process.argv.slice(2), true);
const defaultOptions = parseArgv([]);
const configOptions = parseConfig(cliOptions.opts || defaultOptions.opts);

const options = _.defaults({}, cliOptions, configOptions, defaultOptions);
```